lib/tag-list/index.tsx

```diff
@@ -196,13 +196,13 @@ const mapStateToProps: S.MapState<StateProps> = (state) => {
   const {
     data,
     settings: { sortTagsAlpha },
-    ui: { editingTags, openedTag },
+    ui: { editingTags },
   } = state;
   return {
     editingTags,
+    openedTagHash: selectors.openedTagHash(state),
```

Comment:
it would be preferable to rely on `tagHash` here than creating another selector, since the selectors are best when they are slim and specific to getting data out of state, calculating it _if necessary_

Reply:
Fixed!